.github/workflows/ci.yml

```diff
@@ -99,6 +99,7 @@ jobs:
           destination_path: test/resources/destinations/resource.yaml
           configuration_path: test/resources/configurations/resource.yaml
           configuration_output_dir: test/otel/${{ matrix.bindplane_versions }}
+          configuration_output_branch: otel-raw-configs
```

Comment:
CI for this workflow will output to `otel-raw-configs` branch. I pushed this branch and removed everything from it. Once merged, this branch will populate with the raw configs.